glossary.yml

```diff
@@ -1154,6 +1154,13 @@
       Hoja ni mojawapo ya misemo kadhaa ambayo hupitishwa kwa chaguo la kukokotoa. 
       Ni thamani halisi inayopitishwa. Vigezo na hoja ni tofauti, lakini dhana zinazohusiana. 
       Vigezo ni vigeuzo na hoja ni thamani zilizopewa vigeu hivyo.
+  uk:
+    term: "аргумент"
+    def: >
+      Термін не слід плутати з [параметром](#parameter), це не синонім. Аргумент — це один з можливих 
```

Comment:
English text says "An argument is one of possibly several expressions that are passed to a function" meaning that it may have several arguments. Ukrainian text sounds more like a type of an expression that a function may receive. That sounds misleading.

Reply:
Well... OK